content/800-data-platform/050-data-proxy/500-deploy.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "Deploy with the Data Proxy"
+metaTitle: "Deploy with the Data Proxy"
+metaDescription: "Deploy with the Data Proxy"
+---
+
```

Comment:
This is a weird page. 

There are many more way to deploy a project that uses the Data Proxy. Vercel Serverless Functions and Cloudflare Workers are just 2 platforms where usage of Data Proxy is _necessary_ as you can not deploy the normal Engine there directly.

Long term I think it would be nice to reflect that better and write an introduction how and when deploying with the Data Proxy Client actually makes sense or might be necessary.